phpseclib/Net/SSH2.php

```diff
@@ -4152,4 +4163,42 @@ function setWindowSize($columns = 80, $rows = 24)
         $this->windowColumns = $columns;
         $this->windowRows = $rows;
     }
+
+    /**
+     * @return string
+     */
+    function __toString()
+    {
+        return $this->getResourceId();
+    }
+
+    /**
+     * @return string
```

Comment:
> Explain why the returned string contains { } here.

(I think bantu is wanting you to add a comment in the code explaining {} - not to explain it on github because that's already been done lol)